Tests/XCUITests/NightModeTests.swift

```diff
@@ -12,6 +12,7 @@ class NightModeTests: BaseTestCase {
         mozWaitForElementToExist(app.tables.otherElements[ImageIdentifiers.nightMode])
     }
 
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2307056
```

Comment:
Wondering if it would make sense to move this to Settings too... I'm fine with both options

Reply:
I considered this solution, but technically the option is outside settings.